docker-compose.yml

```diff
@@ -24,4 +24,20 @@ services:
         - 5000:5000
     volumes:
         - ./todo-app:/app/todo_app 
-    privileged: true
\ No newline at end of file
+    privileged: true
+
+
+volumes:
+  mongodb:
+
+services:
```

Comment:
Note that this is overriding the above `services` block - we could add this mongo image under that section and then tweak the `dev` setup to use it